manuscript/ch2/2.4.0.0-create_vagrant_box_from_cloud.txt

```diff
@@ -17,7 +17,7 @@ To use this box run:
 {linenos=off, lang=bash}
 ~~~~~~~
 cd $pve/ariadne/orchestration/vagrant
-vagrant init deb/jessie-amd64 --provider virtualbox
+vagrant init deb/jessie-amd64
 ~~~~~~~
```

Comment:
I believe that but I've also some fuzzy memories about that option. I'm just going through the whole packer build process to double check before merging :-)